common/prometheus-server-pre7/examples/kubernetes-health.alerts.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             context: node
             meta: "{{`{{ $value }}`}} nodes NotReady"
             dashboard: kubernetes-health
-            playbook: docs/support/playbook/kubernetes/k8s_node_not_ready
+            playbook: 'https://operations.global.cloud.sap/docs/support/playbook/kubernetes/k8s_node_not_ready
```

Comment:
Syntax error (also for all other places where the playbook link was previously not quoted).

FYI, in my changeset, I used this command to generate all changes:
```
sed -i 's/playbook: ['\''"]\?\(docs\/.*[^'\''"]\)['\''"]\?/playbook: '\''https:\/\/operations.global.cloud.sap\/\1'\''/' $(rg -l playbook:)
```